unblob/strategies.py

```diff
@@ -71,27 +71,6 @@ def search_chunks_by_priority(  # noqa: C901
     return all_chunks
 
 
-def remove_inner_chunks(chunks: List[Chunk]):
```

Comment:
We still need this function because when formats are not inside the other we predicted with the priority, it can happen that we still found chunks inside another.